src/app/doc-pages/paginator-doc/paginator-doc.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="doc-page-container">
+
+  <h1 class="doc-title">Paginator Component</h1>
+
+  <p class="doc-info">
+    Pagination provides navigation to the page, by sharing content across several separate pages
+    and linking them to the bottom / top of the page. 
+    This way you give the user a sense of control by knowing where they are and they can estimate how much time it will take to find it.
+  </p>
+
+  <div class="doc-info-use">
+    <h2><i class="fal fa-check-circle doc-info-use-icons icon-green"></i> <b>When to use:</b></h2>
+    <p>
+      When there is a lot of content to process and the user is looking for specific information. Often used in conjunction with tables.
```

Comment:
Noe ala: When a table with many rows needs to be dinviden into easier to handle pages